VirtoCommerce.Storefront.Model/Security/UserRegistration.cs

```diff
@@ -10,12 +10,14 @@ public partial class UserRegistration
         public string PhotoUrl { get; set; }
 
         [FromForm(Name = "customer[first_name]")]
+        [Required]
```

Comment:
I suggest marking these fields as required on the theme side in order to do not affect other themes where these fields are optional.